gr-azure-software-radio/python/blob_sink.py

```diff
@@ -50,26 +51,16 @@ class BlobSink(gr.sync_block):
     Retry Total: Total number of Azure API retries to allow before throwing an exception. Higher
         numbers make the block more resilient to intermittent failures, lower numbers help to
         more quickly iteratively debug authentication and permissions issues.
+    SigMF Recording: If True, two files will be created, blobname.sigmf-data and blobname.sigmf-meta.
+        The blobname.sigmf-data will contain the signal, while the blobname.sigmf-meta will contain
+        metadata about the signal, provided as additional block params.
     """
     # pylint: disable=too-many-arguments, too-many-instance-attributes, arguments-differ, abstract-method
     def __init__(self, np_dtype: np.dtype, vlen: int = 1, authentication_method: str = "default",
                  connection_str: str = None, url: str = None, container_name: str = None, blob_name: str = None,
-                 block_len: int = 500000, queue_size: int = 4, retry_total: int = 10):
+                 block_len: int = 500000, queue_size: int = 4, retry_total: int = 10, sigmf: bool = False,
+                 sample_rate: float = 0, center_freq: float = np.nan, author: str = '', description: str = '', hw_info: str = ''):
         """ Initialize the blob_sink block
-
-        Args:
```

Comment:
Note how I removed this block of docs, they don't show up in the docs tab and I don't see why we would want docs to live in 2 spots anyway

Reply:
The Blob blocks documentation show up in the docs tab for me. Are you not seeing documentation for all the blocks? I only had issues with the DIFI docs in the past.

Reply:
I meant that I think it's OK to delete those docs, because they don't show up in the doc tab anyway, and they are redundant imo